README.md

```diff
@@ -70,66 +74,79 @@ To install the necessary dependencies for the project, run the following command
 nix develop
 ```
 
-This command will set up a development environment with all the required tools and libraries specified in the `flake.nix` file.
+This command will set up a development environment with all the required tools and libraries specified in the
+`flake.nix` file.
 
-**Note**: You will need to run `nix develop` in every new terminal session before starting development to make the tools available in that shell. This is because the environment is only active within the current shell session and does not persist across multiple terminal sessions. -->
+**Note**: You will need to run `nix develop` in every new terminal session before starting development to make the tools
+available in that shell. This is because the environment is only active within the current shell session and does not
+persist across multiple terminal sessions. -->
 
 # Usage
```

Comment:
Perhaps the entire "Usage" paragraph should be moved to CONTRIBUTING.md

Reply:
I would even say this: everything except this paragraph(screenshot) should be moved across internal *.md files. In the main file, we’ll leave a message about the team invitation, and later we’ll add a brief but concise description of the project there.

<img width="899" alt="image" src="https://github.com/user-attachments/assets/f1052618-9dcc-422e-ab3f-1ec8f73c9d58">

Reply:
> Perhaps the entire "Usage" paragraph should be moved to CONTRIBUTING.md

My reasoning for keeping them the way they are right now is, that I see the `readme` for any user (technical and non-technical included). Non-technical users won't want to click one more link to read how to run the project or how to fire up the editor. But `contributing` is purely for the technical users, who will not bother clicking on as many links as needed to get the project up and running. The following section in `contributing` has a backlink to `requirements` section in the `readme` just below which, there is the `usage` section. 
[https://github.com/keenawa-co/moss/blob/MOSSMVP-260-Improve-CONTRIBUTING.md/CONTRIBUTING.md#setting-up-your-environment](https://github.com/keenawa-co/moss/blob/MOSSMVP-260-Improve-CONTRIBUTING.md/CONTRIBUTING.md#setting-up-your-environment)

This is just to confirm. Are you sure you want to move the `Usage` section to `contributing`?